metaflow/package.py

```diff
@@ -88,8 +88,6 @@ def _walk(self, root, exclude_hidden=True, suffixes=None):
             # if path and (path[0] == '.' or './' in path):
             #    continue
             for fname in files:
```

Comment:
maybe a slightly safer and less expensive check would be something like:
```
if (fname[0] == "." and fname in suffixes) or (fname[0] != "." and any(fname.endswith(suffix) for suffix in suffixes):
```
This would restrict to including just files like `.env` and not check all random . files.

But am fine with this change as well.

Reply:
going with your suggestion as the initial implementation ended up including arbitrary hidden files with known extensions, e.g. `.hidden.py`

Reply:
Good point -- I hadn't even thought of that.